modules/RTC/RTCUtils.js

```diff
@@ -324,7 +334,8 @@ function getConstraints(um, options) {
                 chromeMediaSourceId: options.desktopStream,
                 maxWidth: window.screen.width,
                 maxHeight: window.screen.height,
-                maxFrameRate: 3
+                maxFrameRate: SS_FRAME_RATE,
+                minFrameRate: SS_FRAME_RATE
```

Comment:
There is currently an unfortunate world where there is new gum flow (adapterjs) and old gum flow (non adapterjs). Most browsers are in new gum flow but some still depend on old. Do your changes affect new gum flow for sure?

Reply:
Good question. Will check it.

Reply:
Actually I tested it on Chrome and FF it appears to work. And now when I'm looking at the adapter we are using it as polyfill, so the new parameters are passed to it.

Reply:
@virtuacoplenny I checked the code again. And as you suspected it seems that for SS we are using always the old flow!

I plan to change:
https://github.com/jitsi/lib-jitsi-meet/pull/711/files#diff-7d21489f2da42a0c08c868715b66dcddL454
Sorry I missed it :(

And then when we fix the bug it should just work :)

Reply:
Okay